src/main/java/com/passpnu/passwordmanager/config/WebSecurityConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.passpnu.passwordmanager.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.authentication.AuthenticationManager;
+
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfiguration{
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception{
+        http.csrf().disable().authorizeHttpRequests(
+                (requests) -> requests.anyRequest().permitAll()
```

Comment:
You compromise your application security by permitting all requests. Think about levels of access your current endpoints should have and permit or require authentication for them accordingly (refer to https://spring.io/guides/gs/securing-web/). (Implement or check if Basic Auth is already implemented)

Reply:
Added some implemented endpoints